mantis-common/src/main/java/io/mantisrx/runtime/descriptor/StageSchedulingInfo.java

```diff
@@ -29,12 +29,13 @@
 
 public class StageSchedulingInfo {
 
-    private int numberOfInstances;
-    private MachineDefinition machineDefinition;
-    private List<JobConstraints> hardConstraints;
-    private List<JobConstraints> softConstraints;
+    private final int numberOfInstances;
+    private final MachineDefinition machineDefinition;
+    private final List<JobConstraints> hardConstraints;
+    private final List<JobConstraints> softConstraints;
     private StageScalingPolicy scalingPolicy;
```

Comment:
why is this not final?

Reply:
There is an existing setter method but I think it's not used. Will remove.